### ISSUE TITLE
Lucky init whines incomprehensibly about Laravel

### ISSUE THREAD

**Issue by BrucePerens** [luckyframework/lucky_cli#251]
Having never had to deal with Laravel before, I don't know if I should be concerned about this or not. However, cybercrud in general trains people to ignore error messages when they get one that actually means something.

```
▸ Installing node dependencies
warning laravel-mix > autoprefixer > browserslist@2.11.3: Browserslist 2 could fail on reading Browserslist >3.0 config used in other tools.
warning laravel-mix > css-loader > cssnano > autoprefixer > browserslist@1.7.7: Browserslist 2 could fail on reading Browserslist >3.0 config used in other tools.
warning laravel-mix > css-loader > cssnano > postcss-merge-rules > browserslist@1.7.7: Browserslist 2 could fail on reading Browserslist >3.0 config used in other tools.
warning laravel-mix > css-loader > cssnano > postcss-merge-rules > caniuse-api > browserslist@1.7.7: Browserslist 2 could fail on reading Browserslist >3.0 config used in other tools.
warning "laravel-mix > img-loader@3.0.0" has unmet peer dependency "imagemin@^5.0.0".

```

**Comment by paulcsmith** [comment]
My guess is the package.json needs to be updated since JavaScript packages seem to be updated every other day. I'll look into it though! I definitely agree the initial setup should be free of warnings

**Comment by paulcsmith** [comment]
Closed by #283
